services/blueprint.yml.erb

```diff
@@ -24,6 +24,7 @@ dashboard_client:
   secret: 'service-fabrik-blueprint-secret'
   redirect_uri: <%= redirect_uri %>
 plan_updateable: true
+actions: Blueprint
```

Comment:
In the blueprint-manifest print out action response. Thats a way to demonstrate how to access action response in manifest.

Reply:
Done